tests/CMakeLists.txt

```diff
@@ -1,28 +1,15 @@
-#  Copyright 2018 U.C. Berkeley RISE Lab
-# 
-#  Licensed under the Apache License, Version 2.0 (the "License");
-#  you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
-# 
-#      http://www.apache.org/licenses/LICENSE-2.0
-# 
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
-
-cmake_minimum_required (VERSION 3.6)
-
-add_executable(run_lattice_test run_test.cpp)
-
-IF(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-   # Mac OS X specific code
-   target_link_libraries (run_lattice_test ${PROJECT_SOURCE_DIR}/vendor/gtest/build/libgtest.dylib)
-ENDIF(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-
-IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
-    # Linux specific code
-   target_link_libraries (run_lattice_test ${PROJECT_SOURCE_DIR}/vendor/gtest/build/libgtest.so)
-ENDIF(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
-
+CMAKE_MINIMUM_REQUIRED(VERSION 3.6)
+
+ADD_EXECUTABLE(run_lattice_test run_lattice_test.cpp)
+TARGET_LINK_LIBRARIES (run_lattice_test ${GTEST_LIBS_DIR}/libgtest.so)
+ADD_DEPENDENCIES(run_lattice_test googletest)
+
+ADD_EXECUTABLE(run_server_handler_test
+			   run_server_handler_test.cpp
+			   mocked.cpp
+			   ../src/bedrock/kvs/node_join_handler.cpp
```

Comment:
I think there's a CMake way to list all the CPP files in a directory with a wildcard. We'll need to include all of them anyway for the tests once we have 'em.

Reply:
let's leave this for now and fix it when we re-organize the directory?